app/src/main/res/values/component_colors.xml

```diff
@@ -186,4 +186,13 @@
   <color name="component_color_math_expression_parser_activity_radio_buttons_color">@color/color_palette_dark_text_color</color>
   <color name="component_color_math_expression_parser_activity_edit_text_color">@color/color_palette_dark_text_color</color>
   <color name="component_color_math_expression_parser_activity_hint_text_color">@color/color_palette_hint_text_color</color>
+    <!-- Audio Language Activity -->
+  <color name="component_color_audio_language_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_language_items_background_color">@color/color_palette_container_background_color</color>
+  <color name="component_color_language_items_heading_text_color">@color/color_palette_dark_text_color</color>
+  <!-- Reading Text Size Activity -->
+  <color name="component_color_reading_text_size_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_reading_text_size_items_heading_text_color">@color/color_palette_dark_text_color</color>
```

Comment:
@rt4914 actually the text color is already White `#FFFFFF` , due to my Phone brightness is low that's why it's looking fade in the screenshot.

For the selected color, `#00A89F` I'm changing that one.

Reply:
I don't think that brightness affects the screenshot. How are you taking screenshots?

Reply:
Simply as by default screenshot option in Android.

Reply:
> 

As it's showing white to me when I'm testing, also in code the color is white and light green for selection.

https://user-images.githubusercontent.com/76530270/180618911-1fd4171a-999a-4c3f-8625-72b9179bc4b0.jpg

Reply:
> 

@rt4914 can you please check this one? Is it okay?

https://user-images.githubusercontent.com/76530270/181081446-24ae39b1-e752-45dc-a2c6-5470881d48c0.jpg

Reply:
Actually it does not look correct still. I will try to run this locally and see why the screenshots are coming in different colors.

Reply:
Now it looks correct.